sw/ground_segment/cockpit/live.ml

```diff
@@ -1260,10 +1260,20 @@ let listen_flight_params = fun geomap auto_center_new_ac alert alt_graph ->
   let get_cam_status = fun _sender vs ->
     let ac = get_ac vs in
     let a = fun s -> PprzLink.float_assoc s vs in
-    let cam_wgs84 = { posn_lat = (Deg>>Rad)(a "cam_lat"); posn_long = (Deg>>Rad)(a "cam_long") }
-    and target_wgs84 = { posn_lat = (Deg>>Rad)(a "cam_target_lat"); posn_long = (Deg>>Rad)(a "cam_target_long") } in
-
-    ac.track#move_cam cam_wgs84 target_wgs84
+    let lats_str = PprzLink.string_assoc "lats" vs in
+    let longs_str = PprzLink.string_assoc "longs" vs in
+    
+    let lats = List.map float_of_string (Str.split list_separator lats_str) in
+    let longs = List.map float_of_string (Str.split list_separator longs_str) in
+
+    let target_wgs84 = { posn_lat = (Deg>>Rad)(a "cam_target_lat"); posn_long = (Deg>>Rad)(a "cam_target_long") } in
+    
+    let geo_1 = { posn_lat = (Deg>>Rad)(List.nth lats 0); posn_long = (Deg>>Rad)(List.nth longs 0) } 
```

Comment:
@gautierhattenberger  In the example you gave me for DL_Settings Arrays were used. I don't think there should be too much performance degradation keeping it in lists. Please correct me if I am wrong. Do i actually need to put exception handling in for List.nth? Will safe_bind catching it and notifying the user be sufficient?

Reply:
Considering the sizes involved, list or arrays should not make a big difference.
The `safe_bind` should catch exceptions and print them.